scripts/configure-multi-node-aio.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
I'm not sure that I understand the value of having this as a different file. The difference between this and https://github.com/rcbops/rpc-openstack/blob/master/scripts/bootstrap-aio.sh is that script partitions the largest secondary disk as far as I can see. Other than that, not much else.